test/test_lldp_unit.cpp

```diff
@@ -0,0 +1,55 @@
+/*********************************************************************
```

Comment:
I think we can put this in the existing test/test_lldp.cpp file

Reply:
ok